contrib-viz/calc-contrib-stats.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# After running this Python script to generate the figures, commit the figures
+# and run the version-figures.sh script to update the CONTRIB_STATS_JSON with the
+# versioned figure URL
+
+echo "Generating contributor statistics and figure"
+jupyter nbconvert --to notebook --execute --ExecutePreprocessor.kernel_name=python3 --output output.ipynb contrib-viz/01.contrib-git.ipynb
+jupyter nbconvert --to notebook --execute --ExecutePreprocessor.kernel_name=ir --output output.ipynb contrib-viz/02.contrib-viz.ipynb
+jupyter nbconvert --to notebook --execute --ExecutePreprocessor.kernel_name=python3 --output output.ipynb contrib-viz/03.contrib-stats.ipynb
+
+# Remove temporary files
+rm contrib-viz/commits.tsv contrib-viz/contributors.tsv
```

Comment:
Do the above commands also create `output.ipynb` that can be removed?  Or do we want to give the outputs unique names and store them?

Reply:
We can remove it, I think. Everything we need is saved as a json or image file.

```suggestion
rm contrib-viz/commits.tsv contrib-viz/contributors.tsv output.ipynb
```